apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/adresse/AdresseService.kt

```diff
@@ -13,8 +14,11 @@ class AdresseService(
     private val navansattKlient: NavansattKlient,
     private val regoppslagKlient: RegoppslagKlient,
 ) {
-    suspend fun hentMottakerAdresse(ident: String): Mottaker {
-        val regoppslag = regoppslagKlient.hentMottakerAdresse(ident)
+    suspend fun hentMottakerAdresse(
+        sakType: SakType,
+        ident: String,
+    ): Mottaker {
+        val regoppslag = regoppslagKlient.hentMottakerAdresse(sakType, ident)
```

Comment:
Kaller denne PDL?

Reply:
Slackposten peikar vidare hit: https://nav-it.slack.com/archives/CDYU0Q86L/p1707298478556129

Som er akkurat samme regoppslag-klient som vi bruker. Det er vel ei PDL-overbygning innad i PO pensjon, så dei kallar vel vidare på PDL. Det samme gjeld potensielt også andre PO person-tenester.

Reply:
> Uheldig om vi har plutselig andre indirekte innganger dit

The fundamental theorem of software engineering:
_"We can solve any problem by introducing an extra level of indirection."_

🤡